sw/device/lib/dif/dif_spi_host.h

```diff
@@ -4,20 +4,196 @@
 
 #ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_SPI_HOST_H_
 #define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_SPI_HOST_H_
-
 /**
  * @file
- * @brief <a href="/hw/ip/spi_host/doc/">SPI Host</a> Device Interface
- * Functions
+ * @brief <a href="/hw/ip/spi_host/doc/">SPI Host</a> Device Interface Functions
  */
 
+#include <stdint.h>
+
 #include "sw/device/lib/dif/autogen/dif_spi_host_autogen.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif  // __cplusplus
 
-// TODO: Add DIFs.
+/**
+ * Runtime configuration for SPI Host.
```

Comment:
Can you drop comments for all of the fields?

Reply:
I think you mean "can I _add_ comments ..." :)